lemminx-liberty/src/main/java/io/openliberty/lemminx/liberty/LibertyHoverParticipant.java

```diff
@@ -41,19 +40,14 @@ public Hover onText(IHoverRequest request) {
 
 		// if we are hovering over text inside a <feature> element
 		if (parentElement.getTagName().equals(LibertyConstants.FEATURE_ELEMENT)) {
```

Comment:
parentElement.getTagName() could be null (when XML is not valid), so I suggest to write LibertyConstants.FEATURE_ELEMENT.equals(parentElement.getTagName())